src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -529,125 +528,109 @@ namespace HVACCooledBeam {
         int OutWaterNode; // unit outlet chilled water node
         Real64 RhoAir;    // air density at outside pressure and standard temperature and humidity
         Real64 rho;       // local fluid density
-        int Loop;         // Loop checking control variable
```

Comment:
[src/EnergyPlus/HVACCooledBeam.cc:532]:(style),[variableScope],The scope of the variable 'Loop' can be reduced.